plugins/windows-debug.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: windows-debug
+spec:
+  version: v0.1.1
+  homepage: https://github.com/jsturtevant/windows-debug
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/jsturtevant/windows-debug/releases/download/v0.1.1/kubectl-windows-debug-v0.1.1.tar.gz
+    sha256: 8d4cf283b80220f8729a0aa759038a52b4a8d46f49c42b1ccabc036b9f92ada9
+    bin: kubectl-windows-debug
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/jsturtevant/windows-debug/releases/download/v0.1.1/kubectl-windows-debug-v0.1.1.tar.gz
+    sha256: 8d4cf283b80220f8729a0aa759038a52b4a8d46f49c42b1ccabc036b9f92ada9
+    bin: kubectl-windows-debug
+  shortDescription:  Launch a Windows host process pod that will give you access to the specified Windows node
+  description: |
+    Usage:
+      kubectl windows debug <node-name>
```

Comment:
we already print this `Usage` string upon installation, avoid. let users call --help and you can show them there. this will go out of date here.

also, I don't think it will be `windows debug`, should be `windows-debug` once installed. we replace `-` with `_` in the filename.

Reply:
> we already print this `Usage` string upon installation, avoid. let users call --help and you can show them there. this will go out of date here.

sounds good.

according to https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/#naming-a-plugin 

>For example, a plugin that wishes to be invoked whenever the command kubectl foo bar baz is invoked by the user, would have the filename of kubectl-foo-bar-baz

I tried it and it worked this way after installing locally via `kubectl krew install --manifest=manifest.yaml --archive=kubectl-windows-debug-v0.1.1.tar.gz`

Reply:
weird. I think I can confidently say that it should not work. we replace the plugins that contain dashes with underscore:

```
ls ~/.krew/bin
kubectl-allctx     kubectl-example  kubectl-images       kubectl-krew  kubectl-neat        kubectl-open_svc   kubectl-rbac_lookup  kubectl-resource_capacity  kubectl-tree              kubectl-who_can
kubectl-ca_cert    kubectl-flame    kubectl-interact_sh  kubectl-kurt  kubectl-nodeinfo    kubectl-print_env  kubectl-rbac_tool    kubectl-resource_view      kubectl-view_allocations  kubectl-whoami
kubectl-eksporter  kubectl-get_all  kubectl-karmada      kubectl-mc    kubectl-oidc_login  kubectl-prof       kubectl-rbac_view    kubectl-tail               kubectl-view_secret
```

and therefore those plugins installed via krew does not work without dashes:

```
$ kubectl oidc login
Error: unknown command "oidc" for "kubectl"
Run 'kubectl --help' for usage.
$ kubectl who can
Error: unknown command "who" for "kubectl"
Run 'kubectl --help' for usage.
```

Perhaps you have another binary in your `%PATH%` with `kubectl-a-b` format.

Reply:
After removing the previously installed binary, found a bug with the file not being marked as executable.  It is update and I bumped the version in the manifest file here